web/src/regression/core.test.ts

```diff
@@ -73,9 +74,11 @@ describe('Core functionality regression test suite', () => {
 
     test('2.2.1 User settings are saved and applied', async () => {
         const getSettings = async () => {
-            await driver.page.waitForSelector('.view-line')
+            await retry(async () => {
+                await driver.page.waitForSelector('.e2e-settings-file * .monaco-editor')
```

Comment:
I think that's what you mean, the `*` in between would mean "there must be an element in between, but it doesn't matter what kind of element" :)

```suggestion
                await driver.page.waitForSelector('.e2e-settings-file .monaco-editor')
```